zebROS_ws/src/pf_localization/src/pf_localization_node.cpp

```diff
@@ -59,18 +59,20 @@ void rotCallback(const sensor_msgs::Imu::ConstPtr& msg) {
 
 void publish_prediction(const ros::TimerEvent &/*event*/)
 {
-  const Particle prediction = pf->predict();
-
-  // Publish message with predicted pose
-  // TODO - see if std ROS odom message is a better fit than a custom messge
-  pf_localization::pf_pose pose;
-  pose.x = prediction.x_;
-  pose.y = prediction.y_;
-  pose.rot = prediction.rot_;
-  pub.publish(pose);
+  const geometry_msgs::PoseWithCovarianceStamped prediction = pf->predict();
+  pub.publish(prediction);
```

Comment:
Nitpicking - the header field of the message should be filled in, either here or in pf->predict.
The header.stamp files can be set to ros::Time::now().  The header.frame_id field can be set to the odom_frame_id global var.